test/concurrency.test.js

```diff
@@ -0,0 +1,72 @@
+
+var queue = require('..')
+var seneca = require('seneca')
```

Comment:
Feel free to use uppercase `Seneca` for all modules... this makes it easier to set your variable later to `seneca` instead of `s`